crates/aranya-client-capi/src/api/defs.rs

```diff
@@ -468,6 +468,40 @@ impl Addr {
     }
 }
 
+/// The name of a permission.
+///
+/// E.g. "CanAssignRole"
+///
+/// Refer to the "Role Management" section of the policy for an exhaustive list.
+#[repr(transparent)]
+#[derive(Copy, Clone, Debug)]
+pub struct Permission(*const c_char);
```

Comment:
Does the capi tooling care if we make all the types in this file `pub(crate)`? 
```suggestion
pub(crate) struct Permission(*const c_char);
```
We don't plan to import this module directly into any other crates other than the capi tooling if it relies on importing the module. If the tooling is simply parsing the AST, then it may not care.

Reply:
> Does the capi tooling care if we make all the types in this file pub(crate)?

Our capi-codegen crate doesn't but cbindgen does